lang/luajit/Portfile

```diff
@@ -31,31 +34,33 @@ patchfiles          powerpc.patch
 
 post-patch {
     reinplace "s|/usr/local|${prefix}|" ${worksrcpath}/etc/luajit.pc
-}
 
-use_configure       no
+    # Remove -march=i686 from CCOPT_x86
+    reinplace -E {s|-march=[a-z0-9_]+||g} ${worksrcpath}/src/Makefile
+
+    # The build process normally expects git and a .git directory to establish
+    # the version but it includes a fallback for a `.relver` file that we use
+    # here
+    system "echo \"${version_date_s}\" > \"${worksrcpath}/.relver\""
```

Comment:
Could you try to create the file with Tcl? For example, https://github.com/macports/macports-ports/blob/7720d5546f4820a9e2c5aa8d4d682dcb57acf29d/math/R/Portfile#L356-L367

Reply:
Thanks for the tip. I was struggling to figure this one out!

https://github.com/macports/macports-ports/compare/2d9cb10fae60075a189491533deadf6a65039456..d6d7f822a5a496424b9c02e7b805d009f1f6a1b6